packages/lit-dev-server/src/middleware/tutorials-middleware.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import type Koa from 'koa';
+
+/**
+ * Koa Middleware to serve '/tutorials/view.html' from '/tutorials/*'.
+ *
+ * @param staticRoot Root of the static server
+ */
+export const tutorialsMiddleware = (): Koa.Middleware => async (ctx, next) => {
+  const path = ctx.request.path;
+
+  /**
+   * We want to intercept /tutorials/* but not /tutorials/ itself or the place
```

Comment:
Very small nit, but I would usually use `// ` comments for an inline comment multi-line comments like this, because I don't think the `/**` jsdoc style comment means anything here, because it's not associated with a symbol like a variable, method etc.

Reply:
oh lol It was the only way I could get the auto-comment-next-line feature to work. Forgot to remove the second `*`

Reply:
Actually I meant multi-line `//` comments.

(This is totally just a style preference, not blocking, but since we started the conversation :))

The pattern I've been following is `/**` style jsdoc comments for annotating an API, and `//` (including multi-line) when annotating implementation.

This seems to match the TypeScript style guide: https://google.github.io/styleguide/tsguide.html#comments-when-calling-a-function. And also I think it also matches we do in most of our other libraries (e.g. see https://github.com/lit/lit/blob/7453e365000e6a289c139cf7e175a4742296333d/packages/lit-html/src/lit-html.ts#L174)